xain_fl/fl/coordinator/controller.py

```diff
@@ -12,51 +12,54 @@ class Controller(ABC):
     enables different selection strategies.
```

Comment:
(this should be at line 10, but github doesn't allow me to pin it there.)

as far as i know there is no name for the `Controller` in the glossary yet, but this name is pretty vague, especially since it is more of a "selector" than a "controller". it would probably be better to align with Michael on the terminology and change this accordingly.

the same holds for its subclasses.

Reply:
totally agree that the naming is a bit off. 
we have a ticket about renaming some stuff: https://xainag.atlassian.net/browse/XP-382 maybe this can be put in there if the scope of 382 is enlarged to improving naming a bit more in general? i'll take it with Michael to align on the name for the Controller. 

for the sake of the scope of this PR, do you agree the naming doesn't block this PR?

Reply:
sure, follow up ticket is ok.